client/js/libs/handlebars/parse.js

```diff
@@ -9,6 +9,9 @@ const findEmoji = require("./ircmessageparser/findEmoji");
 const findNames = require("./ircmessageparser/findNames");
 const merge = require("./ircmessageparser/merge");
 const colorClass = require("./colorClass");
+const emojiMap = require("../simplemap.json");
+const invert = require("lodash/invert");
+const emojis = invert(emojiMap);
```

Comment:
Should we put these in the build rather than having them built from a file in the repo and also stored in the repo? So that every build creates these 2 files new and the only one stored in the repo is the generate emoji file.

Reply:
That would require the build to download the file every time, that's bad.

Reply:
Oh it downloads a file. Didn't realise that. Meh. Can the file be downloaded at npm install time?

Reply:
We could add https://yarnpkg.com/en/package/emojione in our `devDependencies` and generate the map in `public` as part of `yarn build` instead of checking the file in. That's an entire package for a single file, but it's not that bad as it would be in `devDependencies`.
I'm assuming you didn't mean as part of this PR.

Reply:
I mean, just because we are going from having 1 file to having 2 and we are changing what we are building. If you think it should wait, then fine, I was just trying to not increase duplication as part of this pr.

Reply:
Yeah so I totally agree with you that, but unless we're misunderstanding each other, we're still going to have 2 files, both built based on one, and it's going to be the same script that assembles them.

I'm very much in favor improving this but I'd put this in the "Internal" category while @MaxLeiter's attempt solves an actual thing, both for screen readers and for whoever wants to see a description of the emoji.

Reply:
(I'm very pro this PR btw)

Reply:
Totally agree we should put this in our build pipeline, I just didn't want to turn @MaxLeiter's 4 LOC PR into a refactor :D (even though there are already a bit of changes required).

Also, just to clarify in case there is some misrepresentation, I think both files would not be exact duplicate, as one maps emoji to shortnames, the other one maps emoji to (complete) names. But yeah, that's semantic, because I think we agree on the solution anyway.

Reply:
So, what exactly should this PR do? Leave as-is and fix lodash later?

Reply:
@MaxLeiter, update the generate-emoji script to generate a second file where keys are `emoji` and values are `name` (the full one, such as `"grinning face"` and not `shortname` which would be `":grinning:"` before taking out the colons).

Then in your code load that new JSON instead of inverting the map with lodash and use it for the `aria-label` and `title`.